data-raw/standard_format_markdowns/combine_data_and_eda.Rmd

```diff
@@ -0,0 +1,251 @@
+
+<!-- load('../data/all_fish_data.rda') -->
+
+<!-- #all_fish_data <- all_fish_data %>% select(-lat, -lon) -->
+<!-- #usethis::use_data(all_fish_data, overwrite = TRUE) -->
+
+<!-- monthly_fish_data <- all_fish_data %>% -->
+<!--   mutate(year = ifelse(!is.na(date), lubridate::year(date), 2016), -->
+<!--          month = lubridate::month(date)) %>% -->
+<!--   mutate(habitat_type = case_when(watershed == "Merced" ~ "perennial instream", -->
+<!--                                   watershed != "Merced" ~ habitat_type)) %>% -->
+<!--   group_by(watershed, habitat_type, year, month) %>% -->
+<!--   summarise_at(vars(mass, fork_length), .funs = c('min', 'max', 'median'), na.rm = TRUE) %>% -->
+<!--   ungroup() |> glimpse() -->
+
+<!-- # save file for use in R package -->
+<!-- # usethis::use_data(monthly_fish_data) -->
+
+<!-- ``` -->
+
+<!-- ## Environmental Data  -->
+
+<!-- ```{r} -->
+<!-- all_enviro <- preyDataProcessing::all_enviro_data %>% glimpse -->
+
+<!-- ``` -->
+
+# Explore Variables
+
+## Authors
+
+```{r}
+
+prey_agg_by_author <- preyDataProcessing::all_prey_density %>%
+  group_by(author) %>%
+  summarise_at(vars(prey_density), .funs = c('min', 'max', 'median', 'var'), na.rm = TRUE)
+
+kable(prey_agg_by_author, digits = 1, caption = "summary statistics of prey density values across authors included in the prey data package")
+
+```
+
+## Habitat Type
+
+Habitat types included in the prey data package include: 
+
+* Agricultural canal
+* Floodplain
+* Perennial instream
+* Side channel
+
+Many habitat types were assigned based on the sampling locations provided by authors. 
+
+```{r}
+
+ggplot(all_prey_density, aes(x = as.factor(month(date)), y = prey_density)) + 
+  geom_point() + 
+  facet_wrap(~habitat_type) + 
+  xlab('month') +
+  ylab('prey density (count/L)') + 
+  ggtitle('Distribution of all prey density across years and habitat types', 
+          subtitle = "Aggregated data across all authors") 
+
+ggplot() +
+  geom_boxplot(data = monthly_prey_density, aes(x = as.factor(month), y = median_prey_density), alpha = 0.5) +
+  geom_jitter(data = monthly_prey_density, aes(x = as.factor(month), y = median_prey_density, color = habitat_type)) +
+  scale_y_continuous(trans='log10') +
+  xlab('month') +
+  ylab('log(median prey density) (count/L)') +
+  scale_color_brewer(palette = 'Dark2') +
+  coord_flip() +
+  theme_minimal()
+
+preyDataProcessing::all_prey_density %>%
+  group_by(habitat_type) %>%
+  summarise(author = paste0(unique(author), collapse = ", ")) %>% kable()
+
+all_prey_density %>%
+  group_by(habitat_type, author) %>%
+  select(prey_density) %>%
+  summarise_if(is.numeric, .funs = c("mean", "min", "max", "var"), na.rm = TRUE) %>% 
+  kable(digits=1, format.args = list(big.mark = ","))
+
+```
+
+## Watersheds
+
+There are `r length(unique(monthly_prey_density$watershed))` watersheds included in the monthly prey density data, including:
+
+* Butte Creek
+* Feather River
+* Merced
+* North Delta
+* Sacramento River
```

Comment:
Do we know which section of the sit model this links to? Lower Sac, Upper Sac, Upper mid sac or lower mid sac?